app/Http/Controllers/Kits/CheckoutKitController.php

```diff
@@ -33,7 +33,7 @@ public function __construct(PredefinedKitCheckoutService $kitService)
      * Show Bulk Checkout Page
      *
      * @author [D. Minaev.] [<dmitriy.minaev.v@gmail.com>]
-     * @return View View to checkout
+     * @return \Illuminate\Contracts\View\View View to checkout
```

Comment:
This seems a little inconsistent with what was done in some of the other controllers - there, it looked like you pulled the `@return` and here you're namespacing them. Not a big deal, and also not something I'm going to hold up a PR for, but, consistency blah blah. I can imagine that you just didn't want the tighter return-type checking on these methods since they're exercised less? I can imagine that.

Reply:
Again, just oversight. I don't feel it's a merge-blocker tho. It's at least correct now, where it wasn't before.